src/main/scala/is/hail/rvd/OrderedRVDType.scala

```diff
@@ -110,24 +94,19 @@ class OrderedRVDType(
   override def toString: String = {
     val sb = new StringBuilder()
     sb.append("OrderedRVDType{key:[[")
-    partitionKey.foreachBetween(k => sb.append(prettyIdentifier(k)))(sb += ',')
-    sb += ']'
-    val rowRestKey = key.drop(partitionKey.length)
-    if (rowRestKey.nonEmpty) {
-      sb += ','
-      rowRestKey.foreachBetween(k => sb.append(prettyIdentifier(k)))(sb += ',')
+    if (key.nonEmpty) {
```

Comment:
Why is there double brackets? Is it because key == partitionKey now?

Reply:
Right, this is to maintain forward compatibility. Old versions reading files written by this version will see `key == partitionKey`. Maybe this isn't necessary, but for now I tried to minimize disruption to the file format.